### DISCUSSION TITLE
Can I disable any day I want in the DateTimePickerModal?

### DISCUSSION THREAD

**Discussion by FooJiaJieng** [mmazzarolo/react-native-modal-datetime-picker#638] [Q&A] [answered]
I want to disable Sunday or any day in the DateTimePickerModal. Is it possible?

**Comment by mmazzarolo** [comment, marked as answer]
👋 @FooJiaJieng .
Nope, you can't :/ 
See https://github.com/mmazzarolo/react-native-modal-datetime-picker#how-can-i-allow-picking-only-specific-dates
